alerter/ThresholdAlerterStubImpl.java

```diff
@@ -0,0 +1,11 @@
+
+
+public class ThresholdAlerterStubImpl implements ThresholdAlerterStub{
+
+   public Integer networkAlertStub(float celcius) {
```

Comment:
Intention of this session is to keep the production code (i.e. with bugs) as is and define test cases.
test cases should define or express the desired behavior of the production code. As a result the test cases that is added newly will fail due to bugs in the code. Hence failing test cases clearly indicate that there is something wrong in the production code. Please identify and add test cases that defines the output of the production code.

Reply:
There was no logic mentioned in networkAlertStub(), so I assumed celcius > 200 would lead to breach of threshold. Can you please elaborate on the point whether any refactoring is required in networkAlertStub() or I need to write more test cases. If the answer is yes for test cases what edge cases I can consider for the failure output of production code

Reply:
Since we need to practice TDD, we should define what is the expected output before writing production code. Writing test cases first with defined production code APIs (note implementation is not important yet) will help us achieve our goal. 
In this case we know that production code is incomplete or with bugs. So we keep it as is and try to write as many tests as possible and see that the tests fail because production code is not behaving as how we expect it to. 

Let me know if this is not clear yet 👍